pyaedt/edb_core/siwave.py

```diff
@@ -246,6 +246,80 @@ def create_circuit_port_on_pin(self, pos_pin, neg_pin, impedance=50, port_name=N
         circuit_port.negative_node.node_pins = neg_pin
         return self._create_terminal_on_pins(circuit_port)
 
+    def create_port_between_pin_and_layer(
+        self, component_name=None, pins_name=[], layer_name=None, reference_net=None, impedance=50.0
```

Comment:
To avoid issues with the `mutation` of the list.
See the suggestion in the code.
```suggestion
        self, component_name=None, pins_name=None, layer_name=None, reference_net=None, impedance=50.0
```